app/src/main/java/com/jaredrummler/mobiletest/di/NetworkModule.kt

```diff
@@ -0,0 +1,68 @@
+package com.jaredrummler.mobiletest.di
+
+import com.jakewharton.retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import com.jaredrummler.mobiletest.BuildConfig
+import com.jaredrummler.mobiletest.networking.ApiService
+import com.jaredrummler.mobiletest.networking.SeedApi
+import dagger.Module
+import dagger.Provides
+import io.reactivex.Scheduler
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.util.concurrent.TimeUnit
+import javax.inject.Named
+import javax.inject.Singleton
+
+@Module
+class NetworkModule {
+
+  @Provides
+  @Singleton
+  @Named(SCHEDULER_IO)
```

Comment:
It is ok. I just prefer `@Qualifiers` instead of `@Named` for such a global thing